rad/src/types/mod.rs

```diff
@@ -96,19 +104,68 @@ impl RadonTypes {
             RadonTypes::Float(inner) => inner,
             RadonTypes::Integer(inner) => inner,
             RadonTypes::Map(inner) => inner,
+            RadonTypes::RadonError(_) => panic!("`RadonTypes::RadonError` is not operable"),
             RadonTypes::String(inner) => inner,
         }
     }
+
+    /// Decodes `RadonTypes::RadonError` items from `cbor::value::Value::Array` values.
+    pub fn try_error_from_cbor_value(value: &CborValue) -> Result<Self, RadError> {
+        match value {
+            CborValue::Array(error_args) => {
+                if !error_args.is_empty() {
```

Comment:
This check is redundant, if `error_args` is empty, `split_first` will return `None`.

Reply:
True